pages/events/summer-school.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <main class="summer-school-page">
+    <AppPageHeaderWithCard
+      :description="eventData.header.description"
+      :cta="eventData.header.cta"
+      :card-title="eventData.header.cardSectionHeading"
+    >
+      <template slot="title">
+        {{ eventData.header.primaryTitle }}
+        <br>
+        {{ eventData.header.secondaryTitle }}
+      </template>
+      <template slot="card">
+        <EventCard v-bind="eventData.header.card" vertical-layout>
+          {{ eventData.header.card.description }}
+        </EventCard>
+      </template>
+    </AppPageHeaderWithCard>
+
+    <AppMosaicSection
+      class="summer-school-page__section"
+      :title="eventData.mosaic.title"
+      :elements="eventData.mosaic.elements"
+    />
+
+    <section class="summer-school-page__section">
+      <h2 class="copy__title">
+        {{ eventData.agenda.title }}
+      </h2>
+      <client-only>
+        <cv-tabs>
+          <cv-tab :label="eventData.agenda.week1.tabName">
+            <AppDataTableSection
+              class="summer-school-page__section"
+              :section-title="''"
+              :data-table-columns="agendaColumnsDataTable"
+              :data-table-elements="eventData.agenda.week1.tableData"
+            />
```

Comment:
As I understand, we're setting `:section-title="''"` in order to remove the title from the **AppDataTableSection** component.

Since the **AppDataTableSection** component is basically just the **AppDataTable** component plus a title, I think that I'd make more sense to use the **AppDataTable** component here directly, like so:

```vue
            <AppDataTable
              :columns="agendaColumnsDataTable"
              :elements="eventData.agenda.week1.tableData"
            />
```

Some margin may be needed on top.